test/common/integreatly_stages.go

```diff
@@ -0,0 +1,56 @@
+package common
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+)
+
+var (
+	products = []v1alpha1.ProductName{
+
```

Comment:
@CathalOConnorRH Do we also need to check for bootstrap here?

Reply:
Good spot. That actually isn't needed anymore. That was left over from a previous attempt at writing this. This test only tests the stages and not the product status. That's covered in another test.